content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Resource Types
+content_type: reference
+---
+
+<!-- overview -->
+
+This page provides information on the different resource types you can request, built into Kubernetes.
+
+There are several different resource types: `cpu`, `memory`, `storage`, `hugepages-*`, and `ephemeral-storage`.
+
+Resource requests on Pods are an integral part of scheduling and 
+limiting and managing resource consumption in a Kubernetes 
+cluster. 
+See [Kubernetes Scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) and [Requests and Limits](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) for more details.
+
+<!-- body -->
+
+## CPU 
+The `cpu` resource represents CPU in cores and can be specified as either a decimal (e.g. `1`, `0.5`) or as millicores (e.g. `500m`, `1000m`). `0.5` = `500m`.
+
+You can request or limit the `cpu` resource for Pods and their containers, and any for object that embeds a [Pod template](/docs/concepts/workloads/pods/#pod-templates).
+
+When the cpu limit is reached, the Pod is throttled.
+
+See [Meaning of CPU](/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu) for more details on what CPU means in kubernetes.
+
+## Memory
+The `memory` resource represents memory in bytes and is specified with a number and a binary suffix. e.g. `500Mi` = `500MiB` = `500 * 1024 * 1024`. 
+
+You can request or limit the `memory` resource in Pods and their containers, and any object that embeds a Pod template.
+
+In contrast with cpu, when the memory limit is reached, the 
+Pod is killed.
+
+See [Meaning of Memory](/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory) for more details on what memory means in kubernetes.
+
+## Storage 
+The `storage` resource represents volume size in bytes and is specified with a number and a binary suffix. e.g. `500Gi` = `500GiB` = `500 * 1024 * 1024 * 1024`
+
+You can use the `storage` resource in a [PersistentVolumeClaim](/docs/concepts/storage/persistent-volumes/#introduction).
+
+## Huge pages
+
+On Kubernetes v1.14 and newer you can request the `hugepages-*` 
+resource. Huge pages are a Linux-specific feature where the node 
+kernel allocates blocks of memory that are much larger than the 
+default page size.
+
+You cannot overcommit `hugepages-*` resources. This is different from the `memory` and `cpu` resources.
+
+You request huge pages with a combination of two binary
```

Comment:
Referring this as 'a combination' is confusing.
The paragraphs above are an exact copy of https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-types.